### PR TITLE
[tts]Japanese voice improvement

### PR DESCRIPTION
**Pull request by ffff-23**
Japanese voice navigation improvement.

### CODE REVIEW

File: data/sound-strings/ja.json/localize.json [hunk 1]
```diff
@@ -1,17 +1,17 @@
 {
 "make_a_slight_right_turn":"右を維持してください。",
-"make_a_right_turn":"右に曲がります。",
-"make_a_sharp_right_turn":"急に右折。",
+"make_a_right_turn":"右方向です。",
```
Comment (bykoianko) [inline data/sound-strings/ja.json/localize.json:3]: ffff-23 Thank you for the PR. As far as I've heard "右に曲がります。" means "Make a right turn", but variant "右方向です。" is "To right direction". Is not it so? Probably: "右に曲がって" or "右に曲がりなさい"? What do you think?

File: data/sound-strings/ja.json/localize.json [hunk 2]
```diff
@@ -1,17 +1,17 @@
...
+"make_a_sharp_right_turn":"右後ろ方向です。",
```
Comment (bykoianko) [inline data/sound-strings/ja.json/localize.json:4]: "右後ろ方向です。- "Right back direction"? Is it used in navigators in Japan?
As a variant "右に急カーブして。"? What do you think?

File: data/sound-strings/ja.json/localize.json [hunk 3]
```diff
@@ -63,5 +63,5 @@
 "in_1_mile":"一マイル先",
 "in_1_5_miles":"一.五マイル先",
 "in_2_miles":"二マイル先",
-"unknown_camera":"この先にある自動速度違反取締装置"
+"unknown_camera":"この先に取締装置があります。"
```
Comment (bykoianko) [inline data/sound-strings/ja.json/localize.json:66]: Is this variant widespread? What do you think about "前に速度違反自動取締装置があります。" or "前方に速度違反自動取締装置があります。"?

File: data/sound-strings/ja.json/localize.json (hunk 1 above)
Comment (ffff-23) [reply data/sound-strings/ja.json/localize.json:3]: In Japan, `右方向です` is generally used for navigation when turning right.

e.g. https://jpn.pioneer/ja/support/manual/navi/19raku2/10883.php
https://www.clarion.com/jp/ja/manuals/NXV977D/guidance_voice.html#ro_guide_direction

File: data/sound-strings/ja.json/localize.json (hunk 2 above)
Comment (ffff-23) [reply data/sound-strings/ja.json/localize.json:4]: In Japan, `右手前方向`, `斜め右手前方向`, `大きく右方向`, etc. are used for navigation when turning 90 degrees or more.

e.g. https://jpn.pioneer/ja/support/manual/navi/19raku2/10883.php#t10931
https://www.clarion.com/jp/ja/manuals/NXV977D/guidance_voice.html#ro_guide_direction

File: data/sound-strings/ja.json/localize.json (hunk 3 above)
Comment (ffff-23) [reply data/sound-strings/ja.json/localize.json:66]: Thank you. `前方に速度違反自動取締装置があります。` is appropriate.

**Comment by ffff-23** [comment]
#10325

**Pull request closed without merging**